src/components/project/project.rules.ts

```diff
@@ -281,26 +301,46 @@ export class ProjectRules {
             Role.FieldOperationsDirector,
           ],
           transitions: [
-            {
-              to: ProjectStep.PendingFinanceConfirmation,
-              type: TransitionType.Approve,
-              label: 'Approve Project',
-            },
-            {
-              to: ProjectStep.PendingZoneDirectorApproval,
-              type: TransitionType.Approve,
-              label: 'Approve for Zonal Director Review',
-            },
-            {
-              to: ProjectStep.FinalizingProposal,
-              type: TransitionType.Reject,
-              label: 'Send Back for Corrections',
-            },
-            {
-              to: ProjectStep.Rejected,
-              type: TransitionType.Reject,
-              label: 'Reject',
-            },
+            ...(isMultiplication
+              ? [
+                  {
+                    to: ProjectStep.EarlyConversations,
+                    type: TransitionType.Reject,
+                    label: 'Send Back for Corrections',
+                  },
+                  {
+                    to: ProjectStep.PendingFinanceConfirmation,
+                    type: TransitionType.Approve,
+                    label: 'Submit for Finance Confirmation',
+                  },
+                  {
+                    to: ProjectStep.DidNotDevelop,
+                    type: TransitionType.Reject,
+                    label: 'End Development',
+                  },
```

Comment:
Why `DidNotDevelop` here instead of `Rejected`?

Reply:
I reached out to Sheri to confirm the [requirement](https://seed-company-squad.monday.com/boards/3451697530/pulses/5934478741/posts/3100606414)

Reply:
updating to include both steps per product teams feedback